content/tutorials/quickstart/authorize-minter.md

```diff
@@ -0,0 +1,762 @@
+  const client = new xrpl.Client(net)
+  results = 'Connecting to ' + getNet() + '....'
+  document.getElementById('standbyResultField').value = results
+  await client.connect()
+  results += '\nConnected, finding wallet.'
+  document.getElementById('standbyResultField').value = results
+  my_wallet = xrpl.Wallet.fromSeed(standbySeedField.value)
+  document.getElementById('standbyResultField').value = results
+```
+
+Define the AccountSet transaction, setting the `NFTokenMinter` account and the `asfAuthorizedNFTokenMinter` flag.
+
+```javascript
+  tx_blob = {
+    "TransactionType": "AccountSet",    
+    "Account": my_wallet.address,
+```
+
+Set `NFTokenMinter` to the account number of the authorized minter.
+
+```javascript
+    "NFTokenMinter": standbyMinterField.value,
+```
+
+Set the `asfAuthorizedNFTokenMinter` flag (the numeric value is _10_).
+
+```javascript
+    "SetFlag": xrpl.AccountSetAsfFlags.asfAuthorizedNFTokenMinter
+    }
+```
+
+Report progress.
+
+```javascript
+    results += '\n Set Minter.' 
+    document.getElementById('standbyResultField').value = results
+```
+
+Prepare and send the transaction, then wait for results
+
+```javascript
+    const cst_prepared = await client.autofill(tx_blob)
+    const cst_signed = my_wallet.sign(cst_prepared)
+    const cst_result = await client.submitAndWait(cst_signed.tx_blob)
+```
+
+If the transaction succeeds, stringify the results and report. If not, report that the transaction failed.
+
+```javascript
+    if (cst_result.result.meta.TransactionResult == "tesSUCCESS") {
+    results += '\nAccount setting succeeded.'
+    results += JSON.stringify(cst_result,null,2)
+    document.getElementById('standbyResultField').value = results
+    } else {
+    throw 'Error sending transaction: ${cst_result}'
+    results += '\nAccount setting failed.'
+    document.getElementById('standbyResultField').value = results
+    }
+```
+
+Disconnect from the XRP Ledger.
+
+```javascript
+  client.disconnect()
+} // End of configureAccount()
+```
+
+## Mint Other
+
+This revised mint function allows one account to mint for another issuer.
+
+```javascript
+
+// *******************************************************
+// ****************  Mint Other  *************************
+// *******************************************************
+
+async function mintOther() {
+```
+
+Connect to the ledger and get the account wallet.
+
+```javascript
+  results = 'Connecting to ' + getNet() + '....'
+  document.getElementById('standbyResultField').value = results
+  let net = getNet()
+  const standby_wallet = xrpl.Wallet.fromSeed(standbySeedField.value)
+  const client = new xrpl.Client(net)
+  await client.connect()
+```
+Report success
+
+```javascript
+  results += '\nConnected. Minting NFToken.'
+  document.getElementById('standbyResultField').value = results
+```
+      
+This transaction blob is the same as the one used for the previous `Mint Token` function, with the addition of the `Issuer` field.
```

Comment:
I suggest linking back to provide context. Also, it's clearer if the code text matches the function name exactly.

```suggestion
This transaction blob is the same as the one used for the [previous `mintToken()` function](mint-and-burn-nftokens.html#mint-token), with the addition of the `Issuer` field.
```